crates/binding/src/checkpointing_facet.rs

```diff
@@ -0,0 +1,2291 @@
+pub use checkpointing_facet::*;
```

Comment:
I would say not committing these generated rust abi files, they will create tons of conflicts among PRs. Why not just generated them on CI?

Reply:
Addressed in #1268